wagtail/admin/templates/wagtailadmin/shared/buttons.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
+
+export default {
+    title: 'Shared / buttons',
+  };
+
+export const base = (args) => (
```

Comment:
```suggestion
export const all = (args) => (
```

`all` makes more sense here.